test/unit/specs/components-v1/example/ExampleItem.spec.js

```diff
@@ -6,7 +6,7 @@ import Buefy from 'buefy';
 import BH from 'bh';
 import store from '@/store';
 import { shallowMount, createLocalVue } from '@vue/test-utils';
-import ExampleItem from '@/components-v1/example/ExampleItem';
+import ExampleItem from '@/components/example/ExampleItem';
```

Comment:
Move test file too.
The correct path is `test/unit/specs/components/example/ExampleItem.spec.js`.

Reply:
moved